standup/settings.py

```diff
@@ -235,16 +216,15 @@ def path(*paths):
 CSP_FONT_SRC = (
     "'self'",
 )
-CSP_FORM_ACTION = ("'self'", AUTH0_DOMAIN) if AUTH0_DOMAIN else ("'self'",)
-CSP_CONNECT_SRC = ("'self'", AUTH0_DOMAIN) if AUTH0_DOMAIN else ("'self'",)
+CSP_FORM_ACTION = ("'self'", SITE_URL) if SITE_URL else ("'self'",)
+CSP_CONNECT_SRC = ("'self'", SITE_URL) if SITE_URL else ("'self'",)
```

Comment:
You must be right. Can't think of any reason that Auth0 would be involved in CSP with the current implementation. Removal +1.